components/director/internal/error_presenter/presenter.go

```diff
@@ -33,12 +33,12 @@ func (p *presenter) Do(ctx context.Context, err error) *gqlerror.Error {
 	errID := p.uuidService.Generate()
 
 	if found := errors.As(err, &customErr); !found {
-		log.C(ctx).WithField("errorID", errID).WithError(err).Errorf("Unknown error: %v", err)
-		return newGraphqlErrorResponse(ctx, apperrors.InternalError, "Internal Server Error [errorID=%s]", errID)
+		log.C(ctx).WithField("errorID", errID).Errorf("Unknown error: %v", err)
+		return newGraphqlErrorResponse(ctx, apperrors.InternalError, "Internal Server Error [errorID=%s, reason: %s]", errID, err.Error())
```

Comment:
I'm not sure for the additional reason message here? I've added it to prevent future cases similar to "self register" flow where the error is from the external call or the error returned by the code is not "our custom" error(graphql error). Where we can have additional information for the failure.